client/my-sites/themes/themes-search-card/index.jsx

```diff
@@ -25,7 +25,7 @@ const ThemesSearchCard = React.createClass( {
 		site: React.PropTypes.oneOfType( [
 			React.PropTypes.object,
 			React.PropTypes.bool
-		] ).isRequired,
+		] ),
```

Comment:
I think you can shorten this now to

```js
site: React.PropTypes.object
```
since the `bool` `propType` was pretty certainly to allow the `site` object prop to be `false` (when unset). Since we're dropping the `isRequired`, we can drop that too, now.